codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: off
+coverage:
+  status:
+    project:
+      default:
+        target: auto
```

Comment:
I know the number has been adjusted recently, due to coveralls failures. What have we done traditionally?